src/symbolic_execution.rs

```diff
@@ -137,18 +137,14 @@ pub fn symbolic_execution(file_name: &String, function_name: &String) -> Option<
         // TODO: Support other input types
         if input.get_type().to_string().eq("\"i1\"") {
             continue;
-        } else if input.get_type().to_string().eq("\"i32\"") {
+        } else if input.get_type().to_string().starts_with("\"i") {
```

Comment:
Nit: Is it possible to call `is_int_type()` on the result of `get_type()` for consistency with how we check for integer types elsewhere in the code?

Reply:
Done 👌